src/main/java/uk/gov/register/presentation/resource/EntryResource.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.register.presentation.resource;
+
+import io.dropwizard.jersey.caching.CacheControl;
+import org.postgresql.util.PSQLException;
+import uk.gov.register.presentation.DbEntry;
+import uk.gov.register.presentation.dao.Entry;
+import uk.gov.register.presentation.dao.EntryDAO;
+import uk.gov.register.presentation.dao.RecentEntryIndexQueryDAO;
+import uk.gov.register.presentation.representations.ExtraMediaType;
+import uk.gov.register.presentation.view.AttributionView;
+import uk.gov.register.presentation.view.SingleEntryView;
+import uk.gov.register.presentation.view.ViewFactory;
+
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.util.Optional;
+import java.util.function.Function;
+
+@Path("/entry")
+public class EntryResource {
+
+    private final EntryDAO entryDAO;
+    private final ViewFactory viewFactory;
+    private final RecentEntryIndexQueryDAO queryDAO;
+    private final RequestContext requestContext;
+
+
+    @Inject
+    public EntryResource(EntryDAO entryDAO, ViewFactory viewFactory, RecentEntryIndexQueryDAO queryDAO, RequestContext requestContext) {
+        this.entryDAO = entryDAO;
+        this.viewFactory = viewFactory;
+        this.queryDAO = queryDAO;
+        this.requestContext = requestContext;
+    }
+
+    @GET
+    @Path("/{entry-number}")
+    @Produces({ExtraMediaType.TEXT_HTML, MediaType.APPLICATION_JSON})
```

Comment:
I assumed the current representation card covers all the new resource. we can have either on card or one for each new resource.